bitcrowd/bitstyles
# [#749] Update badge components

Fixes #749 

## Changes

- Badges are now more configurable — each color variant has its color & background-color specified explicitly in configuration instead of automatically. Defaults to grayscale.
- Badges now have size variants (base and small in default config), and always display as small on small screens
- Moves stories for Badge component into a `.js` file, and includes them into the mdx file
- Adds a component file for Badge — outputs a simple HTML element, mimicing how a component in e.g. React, web-components would work.
- Reactivates the controls plugin for storybook

## 📸

![Firefox_Screenshot_2023-04-27T12-24-00 785Z](https://user-images.githubusercontent.com/2479422/234861153-1d4f9cfa-b999-42d8-84e6-7217cdb9c4f1.png)

## How to QA Visual changes

- `git checkout main` 
- `git pull` 
- `git checkout feature/749-update-badge-components` 
- `yarn`
- `yarn storybook`
- navigate to Badge component

Check:

- [ ] The appearance matches the zeplin designs

## How to QA code changes


## Preflight checks

PR author to check, and delete if not applicable:
